versioned_docs/version-5/machines.mdx

```diff
@@ -143,6 +143,31 @@ const feedbackActor = interpret(customFeedbackMachine).start();
 // logs 'Doing something custom!'
 ```
 
+:::warningxstate
+
+`machine.provide()` is not imperative but rather returns a new machine that includes the implementation overrides.
+
+:::
```

Comment:
Let's make this wrap the entire section (include the bad example)

Reply:
Done!